docs/configuration.md

```diff
@@ -456,6 +456,29 @@ Apart from these, the following properties are also available, and may be useful
     from JVM to Python worker for every task.
   </td>
 </tr>
+<tr>
+  <td><code>spark.jupyter.eagerEval.enabled</code></td>
+  <td>false</td>
+  <td>
+    Open eager evaluation on jupyter or not. If yes, dataframe will be ran automatically
+    and html table will feedback the queries user have defined (see
+    <a href="https://issues.apache.org/jira/browse/SPARK-24215">SPARK-24215</a> for more details).
```

Comment:
Is it usually we include a JIRA ticket in user document like this?

Reply:
Yea, I felt the same thing too but there were the same few instances in this page.